public/form.js

```diff
@@ -3,3 +3,6 @@ var coFounderInput = document.querySelector('#coFounder-name');
 plusCircle.addEventListener('click', function () {
```

Comment:
this was in an old PR that was approved
and this is the front end we cant use es6 its not supported in all browsers